### PR TITLE
Chore: adds mobile images to pictograms design page

### PR DESCRIPTION
**Pull request by abbeyhrt**
Adds mobile images to the Pictograms - Overview page 

Testing: 

Go to Pictograms 0verview page in mobile view and check to make sure the images have the word mobile in the src (DoDontExample images did not have mobile versions)  and make sure all the images load.

### CODE REVIEW

File: 12_TintsShades_2-mobile_1.svg [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (emyarod) [inline 12_TintsShades_2-mobile_1.svg]: is this file addition intentional?

File: src/pages/elements/pictograms/images/11_Gradients_2-mobile-2.svg [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (emyarod) [inline src/pages/elements/pictograms/images/11_Gradients_2-mobile-2.svg]: is this image being used anywhere?

**Comment by abbeyhrt** [comment]
@alisonjoseph On the pictograms page on the IDL site, there are a couple of places where at desktop view there is one large picture of something and at mobile view that picture is split into two and both are displayed like this: 

desktop: 
Screen Shot 2019-08-16 at 11 26 03 AM

mobile: 
Screen Shot 2019-08-16 at 11 26 44 AM
 
In this PR, i just chose one image of the two to display at mobile, is that an okay solution or should i figure out a way to get both pictures to display at mobile and only the big one to display for desktop?

@emyarod that is why those images are there, I added them in before I realized this issue. They are the other images that should be displayed at mobile view. I can remove them if we decide they aren't necessary.

**Comment by alisonjoseph** [comment]
@abbeyhrt we can just combine the two mobile images into one image. Do you have photoshop?? if not I can do it, or we can ask a designer

**Comment by abbeyhrt** [comment]
Sounds good! I do not have photoshop. Let me gather all the images that would need to be put together, real quick.

**Comment by alisonjoseph** [comment]
is this one done? good to merge??

**Comment by abbeyhrt** [comment]
@alisonjoseph It should be now!

**Comment by alisonjoseph** [comment]
This looks good! going to merge

**Pull request merged**